src/Sylius/Behat/Context/Api/Shop/AccountContext.php

```diff
@@ -189,6 +205,48 @@ public function iShouldBeNotifiedThatElementIsInvalid(): void
         );
     }
 
+    /**
+     * @When I browse my orders
+     */
+    public function iBrowseMyOrders(): void
+    {
+        $this->orderShopClient->index();
+    }
+
+    /**
+     * @When I register with previously used :email email and :password password
+     */
+    public function iRegisterWithPreviouslyUsedEmailAndPassword(string $email, string $password): void
```

Comment:
Couldn't it be extracted to `RegistrationContext`?

Reply:
could be... but if I do that I get many names conflicts, so I left it here.